readme.md

```diff
@@ -1,3 +1,8 @@
+> **Note**
+> This is a fork of [np](https://github.com/sindresorhus/np) which adds support for [pnpm](https://pnpm.io). The maintainers of np [understandably don't want to support every package manager](https://github.com/sindresorhus/np/issues/251#issuecomment-400717095), so that's why this fork exists.
+>
+> Aside from adding support for `pnpm`, this fork is equivalent to the upstream version.
+
```

Comment:
this header notice would need to be removed if we decided to not go with the fork